modin/engines/base/io/column_stores/feather_dispatcher.py

```diff
@@ -47,9 +47,9 @@ def _read(cls, path, columns=None, **kwargs):
 
         Notes
         -----
-        `pyarrow` engine and local files only are supported for now,
+        `PyArrow` engine and local files only are supported for now,
```

Comment:
was this flagged by the script? if it was that's a false positive which we should fix... cc @anmyachev

Reply:
Script skip words only with `"` quotes.

Reply:
What will go wrong if `PyArrow` is used here?

Reply:
Yes, this place flagged by script.
I think Vasily mentioned that originally here was mentioned `pyarrow` module, not a name of `PyArrow` library, but, i think, it is ok to left `PyArrow` here.

Reply:
We can leave it here for now, but ideally we should leave formatted portions of text alone.